### PR TITLE
Add recipe for gNALI

### PR DESCRIPTION
**Pull request by xialiu71**
Add recipe for gNALI (gene nonessentiality and loss-of-function identifier)
----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: recipes/gnali/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - gnali = gnali.gnali:main
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
+  run:
+  - python
+  - pybiomart
```
Comment (npavlovikj) [inline recipes/gnali/meta.yaml]: There is no conda package for `pybiomart` - please add that one first in bioconda.

Comment (xialiu71) [reply recipes/gnali/meta.yaml]: Done, thank you for letting me know!

**Comment by xialiu71** [comment]
@BiocondaBot please add label

File: recipes/gnali/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (marcelm) [inline recipes/gnali/build.sh]: Please remove this build script and add this to the `build:` section in the recipe:
```
script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

**Comment by druvus** [comment]
@BiocondaBot please fetch artifacts

File: recipes/gnali/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.0rc0.dev2" %}
```
Comment (npavlovikj) [inline recipes/gnali/meta.yaml]: The tarball is `0.1.0rc0.dev2`. Is it possible to make this an official release instead?

File: recipes/gnali/build.sh (hunk 2 above)
Comment (xialiu71) [reply recipes/gnali/build.sh]: Done, thank you for looking at my pull request and letting me know!

File: recipes/gnali/meta.yaml (hunk 3 above)
Comment (xialiu71) [reply recipes/gnali/meta.yaml]: ed3afa6 Updated version and SHA256

**Comment by xialiu71** [comment]
@BiocondaBot please fetch artifacts

**Comment by npavlovikj** [comment]
@BiocondaBot please merge

**Comment by npavlovikj** [comment]
Thank you for the package and the updates @xialiu71  - the recipe looks great!

**Comment by xialiu71** [comment]
Thanks @npavlovikj !

File: recipes/gnali/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{% set name = "gNALI" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "649ec1d872442da520ed5ca670a1fe64e3307fc695cce75ba2cdaa8b653af6a3" %}
```
Comment (marcelm) [inline recipes/gnali/meta.yaml]: For the next version bump, I suggest to remove this variable and write the checksum directly below. It’s preferable to use Jinja2 variables only when they’re actually used more than once in the recipe.

**Pull request merged**